week2/2-website/css/style.css

```diff
@@ -1,19 +1,389 @@
+/*=============================================
+=            GLOBAL styles            =
+=============================================*/
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
```

Comment:
Nice use of CSS vars, and good idea to apply something you've learned here.

However it's important to note that these don't always work on every browser, particularly IE11. You can use this website to see what works and what doesn't: https://caniuse.com/#feat=css-variables